vent/gui/__init__.py

```diff
@@ -4,7 +4,48 @@
 
 # python standard libraries
 import os
+from collections import OrderedDict as odict
 from PySide2 import QtGui
+from vent.common.values import ValueName, SENSOR
+from vent.gui import styles
+
+LIMIT_GUI_INSTANCE = True
+"""
+(bool): whether there hsould only be one GUI instance at a time. disabled during testing.
+"""
+
+def limit_gui(limit=None):
+    if limit is None:
+        return globals()['LIMIT_GUI_INSTANCE']
+    else:
+        globals()['LIMIT_GUI_INSTANCE'] = limit
+
+
```

Comment:
This is so meta

Reply:
sorta an anti-pattern for combining a get/set function :p